README.md

```diff
@@ -0,0 +1,60 @@
+# kubeadm-systemd
+
+## Introduction
+
+__kubeadm-systemd__ is a tool for creating a multi-node Kubernetes cluster
+on the single machine, created mostly for developers __of__ Kubernetes.
+
+It aims to be as similar to the solutions recommendend for the production
```

Comment:
I'd add here that the focuses is currently on developers of k8s; leaving room for the focus to broaden later.